sis_provisioner/dao/student.py

```diff
@@ -0,0 +1,51 @@
+from uw_person_client import UWPersonClient
+
+
+class UWHandshakeClient(UWPersonClient):
+
+    """
+    only campus 0 and 1, enrollment status 12 (16 too?), class codes 1,2,3,4,5, and 8, major codes not 0-EMBA, 0-GEMBA
+    don't sync majors for campus 1, college code E (Bothell or Foster)
+    """
+
+    def get_students_for_handshake(self):
+        students = self.get_registered_students(
+            include_employee=False,
+            include_student_transcripts=False,
+            include_student_transfers=False,
+            include_student_sports=False,
+            include_student_advisers=False,
+            include_student_intended_majors=False,
+            include_student_pending_majors=False,
+            include_student_requested_majors=False)
+
+        handshake_students = []
+
+        for person in students:
+            if person.student.class_code not in ['1', '2', '3', '4', '5', '8']: continue
+            if person.student.campus_code not in ['0', '1']: continue
+
+            foster_mba_program = False
+            for major in person.student.majors:
+                if major.major_abbr_code in ['0-EMBA', '0-GEMBA']:
+                    foster_mba_program = True
+                    break
+            if foster_mba_program: continue
```

Comment:
I think here you should just check the `handshake_major` I defined in the comment [below](https://github.com/uw-it-aca/handshake-provisioner/pull/2/files#r918405913). If someone is a double major, but happened to have one of these they would get skipped. Maybe in that case you should just ignore any ['0-EMBA', '0-GEMBA'] major in the `get_handshake_major` method?